git_machete/github.py

```diff
@@ -125,6 +125,23 @@ def get_token_from_file_in_home_directory() -> Optional[str]:
             get_token_from_hub())
 
 
+def __extract_failure_info_from_422(response: Any) -> str:
+    additional_msg: str = ''
+    if response['message'] != 'Validation Failed':
+        if 'Reviews may only be requested from collaborators.' in response['message']:
+            additional_msg = "\nThere are some invalid reviewers in .git/info/reviewers file. " \
```

Comment:
Actually! use `get_git_subpath` rather than `.git/`, the git folder doesn't always need to live under `.git/`

Reply:
(see `git machete help file`)